Vertical.HubSpot.Api/Engagements/IEngagementsApi.cs

```diff
@@ -0,0 +1,39 @@
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Vertical.HubSpot.Api.Data;
+
+namespace Vertical.HubSpot.Api.Engagements
+{
+    public enum ObjectType
+    {
+        Contact, 
+        Company,
+        Deal
+    }
```

Comment:
this should get into its own file, makes it awkward to find otherwise

Reply:
Moved to it's own file